docs/labwc-config.5.scd

```diff
@@ -300,14 +300,14 @@ extending outward from the snapped edge.
 	interactive moves. Default is 1.
 
 *<snapping><preview><enabled>* [yes|no]
-	Show a preview when snaping to a window to an edge. Default is yes.
+	Show a preview when snapping to a window to an edge. Default is yes.
 
 *<snapping><preview><delay><inner>*++
 *<snapping><preview><delay><outer>*
-	Sets the delay to show a preview when snapping a window to each type of edges.
-	Defaults are 500 ms.
-	*inner* edges are edges with an adjacent output and *outer* edges are edges
-	without an adjacent output.
+	Sets the delay to show a preview when snapping a window to each type of
+	edges. Defaults are 500 ms.
```

Comment:
I know that it is not an issue introduced by this PR but would you up for also changing this to `[..] to each type of edge.` (singular).